### DISCUSSION TITLE
Categories Missing from Mobile view

### DISCUSSION THREAD

**Discussion by piliener** [community-scripts/ProxmoxVE#84] [General]
As the title says can't find categories in mobile view on the webpage. Low prio obviously with all the dev time going on right now.

**Comment by MickLesk** [comment]
If you click "Search Scripts..." upper left, then you see all Scripts sorted by category?


![Screenshot_20241105-234719.png](https://github.com/user-attachments/assets/6d1cdaf0-5f6d-4657-bbea-81d89375ad4f)

**Reply by piliener** [reply]
Didn't realize they were sorted by category there my mistake
